frontend/src/components/common/DefaultAvatar.vue

```diff
@@ -17,7 +17,8 @@ export default {
 
     computed: {
         nameLatters() {
-            return this.user.name && this.user.name.substr(0, 2).toUpperCase();
+            return this.user.firstName.substr(0, 1).toUpperCase()
```

Comment:
\``${this.user.firstName[0].toUpperCase()${this.user.lastName[0].toUpperCase();}`\`